OpenRA.Mods.Common/Traits/World/ShroudRenderer.cs

```diff
@@ -234,61 +238,52 @@ Edges GetEdges(PPos puv, Func<PPos, bool> isVisible)
 			return info.UseExtendedIndex ? edge ^ ucorner : edge & Edges.AllCorners;
 		}
 
-		void DirtyCells(IEnumerable<PPos> cells)
+		void WorldOnRenderPlayerChanged(Player player)
```

Comment:
What is the significance of `World` in this? Why not just `OnRenderPlayerChanged`?

Reply:
You tell me. This is just being consistent with the nomenclature inside `RadarWidget`.